tests/chainer_tests/links_tests/model_tests/test_classifier.py

```diff
@@ -1,22 +1,35 @@
 import unittest
 
+
 import mock
 import numpy
 
 import chainer
+from chainer import functions
 from chainer import links
 from chainer import testing
 from chainer.testing import attr
 
 
+class accuracy_with_ignore_label(object):
```

Comment:
I guess you needed to define a callable object instead of a function here because if a function is passed to `testing.parameterize` it would unexpectedly receive the `self` argument, and I think it is OK. But as it looks strange, I suppose you should write a comment about the reason.